### PR TITLE
Fix issue causing WebViewExt to not be removed/cleaned up on iOS

### LINKED ISSUE #61: Isn't destroying
Which platform(s) does your issue occur on?
- iOS
- iOS 13
- emulator 

### Please, provide the following version numbers that your issue occurs with:

- CLI: (run `tns --version` to fetch it)
✔ Getting NativeScript components versions information...
⚠ Update available for component nativescript. Your current version is 6.4.0 and the latest available version is 6.4.1.
✔ Component tns-core-modules has 6.4.1 version and is up to date.
✔ Component tns-android has 6.4.1 version and is up to date.
✔ Component tns-ios has 6.5.0-2020-02-19-163529-01 version and is up to date.

### Please, tell us how to recreate the issue in as much detail as possible.
I was trying to play audio file from a website which was using HTML5 audio tag. When I navigated to another page of my APP audio was keep playing. How can I destroy it properly?

Thanks

### PR DESCRIPTION
**Pull request by heyman**
## PR Checklist

- [x] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.

>- [ ] All existing tests are passing.
>  - [ ] On Android
>  - [ ] On iOS 9
>  - [ ] On iOS 10
>  - [ ] On iOS 11
>  - [ ] On iOS 12+

I was not able to run the test suite using the instructions in the README (got a whole bunch of errors).

- [ ] Tests for the changes are included

## What is the current behavior?

Currently, the `WKWebView` created by `WebViewExt` on iOS will not be destroyed when its Page is removed by backward navigation. 

One can verify this behavior by loading an HTML file/URL that does something every X second using `setInterval()`. For example play a sound (or maybe just trigger an event that causes Nativescript to log something to the console). When the Page is removed the JS within the WebView keeps running.

Also, see my [comment in #61](https://github.com/Notalib/nativescript-webview-ext/issues/61#issuecomment-746995747).

## What is the new behavior?

While I was investigating this issue I found [this](https://developer.apple.com/forums/thread/22795) and [this](https://stackoverflow.com/a/32443423/27406), which led me to try out the changes is this PR which turned out to fix the issue!

Please note that I made my changes directly in the `webview-ext.ios.js` file inside `node_modules/@nota/nativescript-webview-ext`, and I've now copied over the two changed lines (and changing `null` -> `null!`for TypeScript syntax) for this PR. I'm very new to Nativescript, and to save myself from figuring out exactly what `tns plugin install` does (I suspect mostly just `npm install`), I took the liberty to submit this PR without actually testing it since it's a two line fix (I did test it by hot patching the JS file in `node_modules`, just not through the `tns plugin install` flow).

Fixes #61.

### CODE REVIEW

**Comment by heyman** [comment]
I currently use the following monkey patch to fix this issue without using a forked version:

```javascript
const WebViewExt = require("@nota/nativescript-webview-ext").WebViewExt;

class FixedWebViewExt extends WebViewExt {
    disposeNativeView() {
        this.wkWebViewConfiguration.userContentController.removeScriptMessageHandlerForName("nsBridge");
        this.wkWebViewConfiguration = null;
        super.disposeNativeView();
    }
}
Vue.registerElement("WebViewExt", () => FixedWebViewExt);
```

File: src/webview-ext.ios.ts [hunk 1]
```diff
@@ -66,6 +66,8 @@ export class WebViewExt extends WebViewExtBase {
     }
 
     public disposeNativeView() {
+        this.wkWebViewConfiguration.userContentController.removeScriptMessageHandlerForName("nsBridge");
```
Comment (m-abs) [inline src/webview-ext.ios.ts:69]: Nice catch.

Just to make sure, should we call `this.wkWebViewConfiguration.userContentController.removeAllUserScripts()` too?

Comment (heyman) [reply src/webview-ext.ios.ts:69]: Hm, I'm not sure. In my case it was not needed for the WKWebview to be cleaned up (it was enough with this patch).

**Comment by m-abs** [comment]
Thank you very much for the contribution. It has been published with v7.0.2

**Comment by heyman** [comment]
Great! Thank you very much for a very useful plugin :)!

**Pull request merged**